aten/src/ATen/native/LinearAlgebra.h

```diff
@@ -25,5 +25,4 @@ using unpack_pivots_fn = void(*)(
 );
 DECLARE_DISPATCH(unpack_pivots_fn, unpack_pivots_stub);
 
-
```

Comment:
Run `git checkout master -- aten/src/ATen/native/LinearAlgebra.h` to revert changes to this file.

Reply:
Done, thanks!